src/test/java/com/worktracker/unit/service/UserServiceTest.java

```diff
@@ -35,11 +35,22 @@ public void setUp() {
 
     @Test
     public void deleteUserSuccess() {
+        when(userRepository.existsById(1L)).thenReturn(true);
+
         userService.deleteUser(1L);
 
         verify(userRepository, times(1)).deleteById(1L);
     }
 
+    @Test(expected = WorktrackerException.class)
+    public void deleteUserNonExist() {
+        when(userRepository.existsById(2L)).thenReturn(false);
+
+        userService.deleteUser(2L);
+
+        verify(userRepository, times(0)).deleteById(2L);
```

Comment:
This line is redundant, as the above one throws the exception and you are not getting to this line.